src/main/java/com/google/gson/typeadapters/UtcDateTypeAdapter.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright (C) 2011 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This is not packaged for maven so we just include it here. */
```

Comment:
This disclaimer should be more highly visible, distinct from the typically unread boilerplate.  I got sidetracked by the jargon in some comments below, thinking we wrote them.